releases/release-6.0.0-dmr.md

```diff
@@ -0,0 +1,593 @@
+
+    [User document](/dm/dm-webui-guide.md)
+
+- Add an error handling mechanism
+
+    More commands are introduced to address problems that interrupt a migration task. For example:
+
+- In case of a schema error, you can update the schema file by using the `--from-source/--from-target` parameter of the binlog-schema update, instead of editing the schema file separately.
+- You can specify a binlog position to inject, replace, skip, or revert a DDL statement.
+
+    [User document](/dm/dm-manage-schema.md)
+
+- Support full data storage to Amazon S3
+
+    When DM performs all or full data migration tasks, sufficient hard disk space is required for storing full data from upstream. Compared with EBS, Amazon S3 has nearly infinite storage at lower costs. Now, DM supports configuring Amazon S3 as the dump directory. That means you can use S3 to store full data when you perform all or full data migration tasks.
+
+    [User document](/dm/task-configuration-file-full.md#task-configuration-file-template-advanced)
+
+- Support starting a migration task from specified time
+
+    A new parameter `--start-time` is added to migration tasks. You can define time in the format of '2021-10-21 00:01:00' or '2021-10-21T00:01:00'.
+
+    This feature is particularly useful in scenarios where you migrate and merge incremental data from shard mysql instances. Specifically, you do not need to set a binlog start point for each incremental migration task. Instead, you can complete an incremental migration task faster by using the `--start-time` parameter in `safe-mode`.
+
+    [User document](/dm/dm-create-task.md#flags-description)
+
+### TiDB Lightning
+
+- Maximum tolerable error
+
+    Added a configuration item `max-error`. The default value is 0. When the value is greater than 0, the max-error function is enabled. If an error occurs in a row during encoding, a record containing this row is added to `type_error_v1` and this row is ignored. When rows with errors exceed the threshold, TiDB Lightning exits immediately.
+
+    Matching the `max-error` configuration, the `lightning_task_info` configuration item records the name of the database that reports a data saving error.
+
+    This feature does not cover all types of errors, for example, syntax errors are not applicable.
+
+    [User document](/tidb-lightning/tidb-lightning-error-resolution.md#type-error)
+
+### Data sharing and subscription
+
+- Support replicating 100K tables simultaneously
+
+    By optimizing the data processing flow, TiCDC reduces the resource consumption of processing incremental data for each table, which greatly improves the replication stability and efficiency when replicating data in large clusters. The result of an internal test shows that TiCDC can stably support replicating 100,000 tables simultaneously.
+
+    [User document]()
```

Comment:
This is missing a link

Reply:
> This is missing a link

Thank you for your reminder. This part does not have a corresponding document, so I will remove this line.